BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -38,6 +38,7 @@ IMPROVEMENTS / FIXES
 		- AI can now use dumbfired missiles (Targeting Mode = None / Radar/Laser missile without the necessary sensor), if no better option available.
 		- Add new 'radarTimeout' field to missile configs to permit dumbfired inertial guidance LOAL radar missiles.
 		- AI can now attampt to use guided bombs against air targets if engage Air = true and nothing else available.
+		- Add new 'boosterMass' and 'sustainerMass' Fields to missile config that allow missiles to have a Fuel usage that can affect mass when  'useFuel' field is true.
```

Comment:
This should be in the latest `IMPROVEMENTS / FIXES` section of the changelog, not the v1.6.1.0 section. Merging in the latest from the dev branch should help with that.